generated/api/v1/notifier_service.swagger.json

```diff
@@ -421,13 +407,27 @@
           "type": "string"
         },
         "credentials": {
-          "$ref": "#/definitions/AWSSecurityHubCredentials"
```

Comment:
curious where those changes are coming from, but its not a blocker for the merge of this PR.

Reply:
Didn't manage to find the root cause, but when this change is not included CI complains about ungenerated files. So I guess I'll just leave it in.